docker-services.yml

```diff
@@ -21,8 +21,8 @@ services:
       - "INVENIO_CELERY_BROKER_URL=amqp://guest:guest@mq:5672/"
```

Comment:
The format is the following 'amqp://myuser:mypassword@localhost:5672/myvhost' so the username and password are guest, as told also [here](https://github.com/inveniosoftware/training/tree/v3.1/03-infrastructure-tour#step-5-access-the-message-queue-rabbitmq). These variables should be added in the .env file

Reply:
Ok, let's try to change those